### DISCUSSION TITLE
Add a flag for deploying without building

### DISCUSSION THREAD

**Discussion by vporton** [dfinity/sdk#3398] [Feature Requests]
I deploy canisters this way:
```sh
dfx deploy pst --argument ...
dfx deploy CanDBIndex --argument ...
dfx deploy NacDBIndex --argument ...
#dfx deploy CanDBIndex
#dfx deploy pst
#dfx deploy payments
dfx deploy main
```
(I commented out deployment that is done by `dfx.json` dependencies instead of deploying by an explicit specifying the canister as `deploy` argument.)

Every `deploy` starts a build process that takes much time. Please, add a flag to deploy that will only deploy (without building) to speed-up this process.

**Comment by viviveevee** [comment]
`dfx deploy` is basically `dfx canister create` + `dfx build` + `dfx canister install`. If you just run `dfx canister install` on your own you get that functionality already
